easybuild/easyconfigs/r/RDFlib/RDFlib-4.2.2-foss-2019b.eb

```diff
@@ -0,0 +1,29 @@
+easyblock = 'PythonBundle'
+
+name = 'RDFlib'
+version = '4.2.2'
+
+homepage = 'https://github.com/RDFLib/rdflib'
+description = """RDFLib is a Python library for working with RDF, a simple yet powerful language
+ for representing information."""
+
+toolchain = {'name': 'foss', 'version': '2019b'}
```

Comment:
@migueldiascosta @lexming rightfully pointed out (via Slack) that we can pull this down to `GCCcore`, so I did that in #9975...